.gitleaks.toml

```
@@ -0,0 +1,16 @@
+title = "Gitleaks custom rules"
+
+[extend]
+# useDefault will extend the base configuration with the default gitleaks config:
+# https://github.com/zricethezav/gitleaks/blob/master/config/gitleaks.toml
+useDefault = true
+
+[allowlist]
+description = "global allow lists"
+paths = [
+    '''gitleaks.toml''',
```

should this be `.gitleaks.toml`? 
```suggestion
    '''.gitleaks.toml''',
```

---

Technically yes but I think its a fuzzy match. Will update to stop any eye twitching in the future 😁